src/Microsoft.AspNetCore.Mvc.DataAnnotations/Internal/DataAnnotationsMetadataProvider.cs

```diff
@@ -80,23 +95,18 @@ public void CreateDisplayMetadata(DisplayMetadataProviderContext context)
                 displayMetadata.DataTypeName = DataType.Html.ToString();
             }
 
-            // Description
-            if (displayAttribute != null)
-            {
-                displayMetadata.Description = () => displayAttribute.GetDescription();
-                displayMetadata.Placeholder = () => displayAttribute.GetPrompt();
-            }
-
             // DisplayFormatString
             if (displayFormatAttribute != null)
             {
                 displayMetadata.DisplayFormatString = displayFormatAttribute.DataFormatString;
             }
 
-            // DisplayName
+            // Display
```

Comment:
These comments are about the properties being set, not the attribute. Let the compiler reorder and merge things if it likes but leave the repeated `displayAttribute != null` checks alone. If anything, move the `displayMetadata.Placeholder` setting into its own block and give it a `// Placeholder` comment.